parser/src/cfg/defcfg.rs

```diff
@@ -338,6 +338,34 @@ pub fn parse_colon_separated_text(paths: &str) -> Vec<String> {
     all_paths
 }
 
+#[cfg(any(target_os = "linux", target_os = "unknown"))]
+pub fn parse_linux_dev(val: &SExpr) -> Result<Vec<String>> {
+    Ok(match val {
+        SExpr::Atom(a) => parse_colon_separated_text(a.t.trim_matches('"')),
+        SExpr::List(l) => {
+            let r: Result<Vec<String>> =
```

Comment:
This could reuse `collect_strings`:

 https://github.com/jtroo/kanata/blob/main/parser/src/cfg/mod.rs#L1805

It doesn't seem necessary to block nested lists

Reply:
I guess collect_strings relies on a `ParsedState` though..

Reply:
> It doesn't seem necessary to block nested lists

Nested list support is a thing mainly because of variables, but variables aren't even available in defcfg.